README.md

```diff
@@ -0,0 +1,44 @@
+skitter_api is a small wrapper around a set of [FlareSolverr](https://github.com/FlareSolverr/FlareSolverr) instances for load
```

Comment:
Would you please wrap this, or use the appendix style links.

While you're in here, you might want to add links to:
* gluetun: https://github.com/qdm12/gluetun
* python-weaver: https://github.com/FanFicDev/python-weaver

We probably don't need both a `README.md` and `README`, please remove `README`.

Did you setup `skitter_api` yourself? Do you think any of these instructions need more context? They assume some knowledge for sure.

Reply:
- Would you please wrap this, or use the appendix style links.
     - Done. I used footnotes
-  While you're in here, you might want to add links...
     - Done.
- We probably don't need both a README.md and README, please remove README
     - Done.
-  Did you set up skitter_api yourself? Do you think any of these instructions need more context? They assume some knowledge for sure.
    - No I did not. I'm not all that familiar with docker, nginx, SQL or headless browsers. I have no experience with any of these tools so I'm not the best person to answer this.

Reply:
Thank you!

I probably used the wrong term, it seems like what I was talking about are "reference-style links": https://www.markdownguide.org/basic-syntax/#reference-style-links where the there's still a clickable link in the main body. Like this:

```markdown
skitter_api is a small wrapper around a set of [FlareSolverr] instances for load
...snip...

[FlareSolverr]: https://github.com/FlareSolverr/FlareSolverr
```

but I can make that change afterward if you don't want to fuss with it more (line 15 is still long and there's some spurious whitespace still which I can also address).

> No I did not. I'm not all that familiar with docker, nginx, SQL or headless browsers. I have no experience with any of these tools so I'm not the best person to answer this.

No worries; I'll update it as I run across people with specific questions.

Reply:
- I probably used the wrong term, it seems like what I was talking about are "reference-style links": https://www.markdownguide.org/basic-syntax/#reference-style-links where the there's still a clickable link in the main body. 
  - Done

Reply:
Thank you!